### PR TITLE
Adding code for SAS disks

### PR DESCRIPTION
**Pull request by COW-Koetje**
As the "Current Drive Temperature:" value is often used on SAS disk I added this to the script to support my SAS disks.

### CODE REVIEW

File: rrd-lib.sh [hunk 1]
```diff
@@ -33,6 +33,12 @@ get_devices () {
     DevTemp=$(/usr/local/sbin/smartctl -a /dev/"${i}" | awk -f $GETTEMP)
     if [ -n "$DevTemp" ]; then
       drivedevs="${drivedevs} ${i}"
+    else
+      #In case the disk is a SAS/SCSI disk without the default number pattern for Smart output
+      DevTemp=$(/usr/local/sbin/smartctl -a /dev/"${i}" | grep "Current Drive Temperature:" |awk '{print $4}')
+      if [ -n "$DevTemp" ]; then
+        drivedevs="${drivedevs} ${i}"
+      fi
       [ -n "$verbose" ] && echo "drivedevs: ${drivedevs}"
```
Comment (seren) [inline rrd-lib.sh]: This verbose/debug output (line 42) should occur after each `drivedevs` update. Could you place this verbose line after line 35 and after line 40?

**Review by seren — CHANGES_REQUESTED** [review]
Thanks for the PR! This looks good except for the one change I noted.

**Comment by COW-Koetje** [comment]
I changed it. 
Although i tested it and it seemed to work with verbose. 
But  you are right this looks way more clean.

**Comment by seren** [comment]
Thanks! Merged.

**Pull request merged**